src/pages/home/home.ts

```diff
@@ -218,11 +219,34 @@ export class HomePage {
     var newMember = {name: nameIn, email: emailIn, role: roleIn};
     this.members.push(newMember);
     this.assForm.teamMembers.push(newMember);
+
+		var name = <any>(document.getElementById("memName"));
+		name.value = "";
+		var email = <any>(document.getElementById("memEmail"));
+		email.value = "";
+		var role = <any>(document.getElementById("memRole"));
+		role.value = "";
+		this.presentToast();
   }
 
-  removeMember(){
-    this.members.pop();
-    this.assForm.teamMembers.pop();
+	presentToast() {
+	  let toast = this.toastCtrl.create({
+	    message: 'Member added to assessment and emailed',
```

Comment:
@dgonca Can we change this a bit -- have the position on the top (on my screen it comes up right in the middle of the input boxes). and maybe `emailed on assessment creation`